codice/ddf-ui
# Update search location drawing tools for 3D map

* Search location shapes can now be edited on the 3D map.
* A new drawing toolbar is now displayed at the top of the map. It's the same toolbar used on the 2D map.

### Testing Instructions
* Test both the basic and advanced search menus.
* Test drawing and editing all 4 shapes (polygon, bounding box, line, point-radius).
  * Drawing: draw a completely new shape.
  * Editing: drag the points/boundaries of an existing shape, and add and remove points.
    * Line: add new points by clicking and dragging the 'plus' icons, move any points on the line, and right click points to remove them.
    * Point-radius: change the radius by moving the circle's edge.
    * Bounding box: move any of the corners.
    * Polygon: add new points by clicking and dragging the 'plus' icons, move any points on the line, and right click points to remove them.
* Test entering shape coordinates manually, then editing the resulting shape.
* Test drawing and editing shapes that cross the antimeridian.
* Test cancelling the drawing via the cancel button on the toolbar and the draw button on the search menu that is toggled to a cancel button when a drawing is in progress.
* Perform some basic testing on the 2D map to ensure drawing was not broken.
